build.sbt

```diff
@@ -1,18 +1,18 @@
 name := "spark-streaming"
 
-version := "0.1"
+version := "0.2"
 
-scalaVersion := "2.12.10"
+scalaVersion := "2.13.12"
 
-val sparkVersion = "3.0.2"
-val postgresVersion = "42.2.2"
-val cassandraConnectorVersion = "3.0.0" // preview version at the moment of writing (July 7, 2020)
-val akkaVersion = "2.5.24"
-val akkaHttpVersion = "10.1.7"
+val sparkVersion = "3.5.0"
+val postgresVersion = "42.6.0"
+val cassandraConnectorVersion = "3.4.1" // preview version at the moment of writing (October, 2023)
+val akkaVersion = "2.8.0"
+val akkaHttpVersion = "10.5.0"
```

Comment:
same for this one, I don't remember which was the last Akka HTTP version that was still OSS